src/lib/testrunner.js

```diff
@@ -422,7 +423,7 @@ class WorkerTestRunner {
         this._currentResult().time = time.getSeconds();
         this.testResults.passes++;
 
-        console.log(green(`      ✔  Succeeded.`), yellow(time.toString()));
+        console.log(green(`      ✔  Succeeded.`), yellow(time.toString()), `(worker ${yellow(this._currentProcessingTime.toString())})`);
```

Comment:
Should the message be more descriptive? or are we trying to save terminal screen space

Reply:
Yes, save space is good, makes it easier to read. (Since it repeats for every test case).

I was thinking this is intuitive enough?

```
... 0s 186ms (worker 0s 180ms)
```

Reply:
We can document it...